### ISSUE TITLE
Odd Lab and Inventory Clear

### ISSUE THREAD

**Issue by DerpKingShay** [Wynncraft/Issues#1844]
Ok, so it's Double XP weekend. The players who never play get on. The servers get laggy as hell. I try to get on, doesn't work. When I finally do get on, I'm in Troms, next to the Emerald Merchant. Then I randomly get teleported to this random place far far outside the map. Some kind of large hall with lots of test chambers. Obviously, I'm confused as hell. Then I log off, and come back later. My inventory is gone. EVERYTHING IS GONE. All I have is the Character Info compass, Quest Book, 14 SP, 6EB and a creeper mask in my inv. That's all. My armor is still there. So I'm like, oh what did I really lose? The rare potato, Relic Daggers, powders, all renewable stuff. A couple of Zhight Potions too, but I don't really care. Then I do /kill, because I'm still stuck in that random test lab thing. Then that's when I realize

Where's my 40/40 white horse?
Please, PLEASE HELP ME.
IGN: ShayRide

**Comment by Xenon2014** [comment]
maybe post some screenshots

**Comment by DerpKingShay** [comment]
Oh yeah... that's... that's smart...
Unfortunately, I don't have any screenshots of my white horse. So that really sucks.
But anyway, here's some screenshots of that random place I got teleported to. I'm not lying about anything, I swear. I did randomly get teleported, I DO have a fully-trained white horse, and my inventory DID get cleared.
![2015-04-26_09 38 41](https://cloud.githubusercontent.com/assets/11828532/7341189/c7c62be0-ec51-11e4-90a9-8ecf1699ed34.png)
![2015-04-26_09 38 34](https://cloud.githubusercontent.com/assets/11828532/7341190/c9874bd0-ec51-11e4-9ac1-b94fa9999f59.png)
![2015-04-26_09 38 54](https://cloud.githubusercontent.com/assets/11828532/7341191/ce3226aa-ec51-11e4-975f-662fe5798604.png)
![2015-04-26_09 38 51](https://cloud.githubusercontent.com/assets/11828532/7341195/d8d15e1e-ec51-11e4-8229-e45f9250e162.png)

**Comment by epicman739** [comment]
Sometimes lag spikes do cause random teleportations. Just remember /kill, and you should be fine.
However, Wynncraft unfortunately has a no refunds policy, so the mods can't help you with your lost items. /fixquests will return missing quest items though.

**Comment by ThunderRemix** [comment]
epicman739, the person said (Then I do /kill, because I'm still stuck in that random test lab thing.)

My guess is that this place has a "town" region that is preventing him or her from being teleported to an actual city. Sadly, Wynncraft has a strict no refunds policy, so you might not get your stuff back. If region inventories are a thing, you MIGHT get your stuff back when you go to out of there, but that's unlikely. A good idea would be to contact a mod or admin to assist you.

**Comment by DerpKingShay** [comment]
Well, I guess I'm kind of over the horse. After using my black horse again, I honestly saw no difference. It felt just as fast, and I don't give a shit about vanity.
ThunderRemix, I'm no longer in that area. I guess my wording in that sentence may have been off-setting. When I did /kill, I spawned in Lusuco.
There was signs on every test tube, it was just some label. I'm pretty sure that cactus one had something about cactus growth. Nothing important.
